po/es.po

```diff
@@ -8372,71 +8372,91 @@ msgid ""
 "types used by Rust library and programs. This way, two libraries can work "
 "together smoothly because they both use the same `String` type."
 msgstr ""
+"Rust viene con una biblioteca estándar que ayuda a establecer un conjunto de "
+"tipos comunes que se usan en la biblioteca y los programas de Rust. De esta "
+"forma, dos bibliotecas pueden funcionar juntas sin problemas, puesto que "
+"ambas utilizan el mismo tipo `String`."
 
 #: src/std.md:7
 msgid "The common vocabulary types include:"
 msgstr ""
+"Entre los tipos de vocabulario más habituales se incluyen los siguientes:"
 
 #: src/std.md:9
 msgid ""
 "[`Option` and `Result`](std/option-result.md) types: used for optional "
 "values and [error handling](error-handling.md)."
 msgstr ""
+"Tipos [`Option` y `Result`](std/option-result.md): se utilizan para valores "
+"opcionales y [gestión de errores](error-handling.md)."
 
 #: src/std.md:12
 msgid "[`String`](std/string.md): the default string type used for owned data."
 msgstr ""
+"[`String`](std/string.md): el tipo de cadena predeterminado que se usa para "
+"los datos propios."
 
 #: src/std.md:14
 msgid "[`Vec`](std/vec.md): a standard extensible vector."
-msgstr ""
+msgstr "[`Vec`](std/vec.md): un vector estándar extensible."
 
 #: src/std.md:16
 msgid ""
 "[`HashMap`](std/hashmap.md): a hash map type with a configurable hashing "
 "algorithm."
 msgstr ""
+"[`HashMap`](std/hashmap.md): un tipo de mapa hash con un algoritmo cifrado "
```

Comment:
```suggestion
"[`HashMap`](std/hashmap.md): un mapa hash con un algoritmo hash configurable."
```

(Cifrado means encryption - I get the intended meaning, but doesn't work that way in Spanish)